### PR TITLE
App 824 token percentage

### PR DESCRIPTION
**Pull request by WissenIstNacht**
## Description

This PR targets the second part of APP-710. It implements the logic to compute the percentage share of each address w.r.t. to the new token amounts after minting.

Task: [823](https://aragonassociation.atlassian.net/browse/APP-824)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

**Review by RakeshUP — APPROVED** [review]
Apart from the lint errors, it's good to go!

File: packages/web-app/src/containers/actionBuilder/mintTokens/addressTokenRow.tsx [hunk 1]
```diff
@@ -92,43 +94,45 @@ const TokenField: React.FC<IndexProps> = ({actionIndex, fieldIndex}) => {
   );
 };
 
-const DropdownMenu: React.FC<AddressAndTokenRowProps> = ({
-  fieldIndex,
-  onDelete,
-}) => {
-  const {t} = useTranslation();
+const DropdownMenu: React.FC<Omit<AddressAndTokenRowProps, 'newTokenSupply'>> =
```
Comment (sepehr2github) [inline packages/web-app/src/containers/actionBuilder/mintTokens/addressTokenRow.tsx]:
``````````js
const DropdownMenu: React.FC<
  Omit<AddressAndTokenRowProps, 'newTokenSupply'>
> = ({fieldIndex, onDelete}) => {
  const {t} = useTranslation();

  return (
    <Dropdown
      align="start"
      trigger={
        <ButtonIcon mode="ghost" size="large" icon={<IconMenuVertical />} />
      }
      sideOffset={8}
      listItems={[
        {
          component: (
            <ListItemAction title={t('labels.removeWallet')} bgWhite />
          ),
          callback: () => {
            onDelete(fieldIndex);
          },
        },
      ]}
    />
  );
};

Comment (sepehr2github) [reply packages/web-app/src/containers/actionBuilder/mintTokens/addressTokenRow.tsx]: Replace your function with this one to remove lint errors

**Review by sepehr2github — APPROVED** [review]
Thanks, Vali. Seems good to me!

File: packages/web-app/src/containers/actionBuilder/mintTokens/addressTokenRow.tsx (hunk 1 above)
Comment (WissenIstNacht) [reply packages/web-app/src/containers/actionBuilder/mintTokens/addressTokenRow.tsx]: looool, my auto-formatter doesn't agree 😂 👍🏼

**Pull request merged**